v1/pipeline/steps/cd/k8s-rolling-step-node.yaml

```diff
@@ -1,55 +1,40 @@
 title: K8sRollingStepNode
 type: object
 required:
```

Comment:
id and name are required by default now? is that why we are removing from here?

Reply:
id and name are optional fields in the V1 schema.